### DISCUSSION TITLE
Twitter old logo in profile

### DISCUSSION THREAD

**Discussion by DylanBT928** [community/community#104722] [Mobile]
### Select Topic Area

Show & Tell

### Body

Twitter, now X, has changed their logo. On the GitHub website, linking your X account shows the new X logo next to your linked account. However, on mobile, it still shows the old Twitter bird logo.

## Mobile preview:

![Mobile](https://github.com/community/community/assets/75633077/ff7bb263-92a3-45b9-9249-5985abc5e911)

## Website preview:

![Website](https://github.com/community/community/assets/75633077/9b90ab03-920b-4019-8f24-bb7e3051664c)

**Comment by elisealix22** [comment]
Thank you for reporting this and for providing the mock, @DylanBT928 ! I opened an internal issue to track this. Please keep the feedback coming.
